docker-compose.yml

```diff
@@ -83,10 +83,9 @@ services:
   redis:
     container_name: redis
     # https://hub.docker.com/_/redis/
-    image: redis:6.2.11-alpine
+    image: redis:7.0.11-alpine
```

Comment:
What about the client lib requirement ?

Reply:
I rebased over https://github.com/mozilla/treeherder/pull/7725, which upgrades `django-redis`. \
Everything seems to work fine with those versions (concerning pulse events).